icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -1482,10 +1478,8 @@ class ConflictType(Enum):
             A zarr metadata update conflicts with an existing zarr metadata update
         ZarrMetadataUpdateOfDeletedArray: int
             A zarr metadata update is attempted on a deleted array
-        UserAttributesDoubleUpdate: int
-            A user attribute update conflicts with an existing user attribute update
-        UserAttributesUpdateOfDeletedNode: int
-            A user attribute update is attempted on a deleted node
+        ZarrMetadataUpdateOfDeletedGroup: int
```

Comment:
these should all. be `tuple[int]` but why that choice?

Reply:
I have no idea actually ... @mpiannucci maybe?